.changeset/red-ducks-battle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-test-data/plain-enum-value': minor
```

Comment:
Does this changeset have to be regenerated now that the directory name has changed? Honest question, I'm not sure :)

Reply:
Package name matters here. What has changed?

Reply:
It has to change as pkg name changed.

Reply:
Yes, slight whiff there on my part.